src/frontend/screens/Security/index.tsx

```diff
@@ -66,12 +67,12 @@ const m = defineMessages({
 
 export const Security: NavigationStackScreenComponent = () => {
   const { passIsSet } = React.useContext(SecurityContext);
-  const [{ experiments }] = React.useContext(SettingsContext);
   const { navigate } = useNavigation();
+  const { appPasscode } = devExperiments;
 
   React.useEffect(() => {
-    if (!experiments.appPasscode) navigate("Settings");
-  }, [experiments]);
+    if (!appPasscode) navigate("Settings");
+  }, [appPasscode]);
```

Comment:
similar comment about not needing to specify `appPasscode` in the deps array, since it's a stable constant